### PR TITLE
[improvement]Remove the website build framework from the docs directory

### LINKED ISSUE #11080: [improvement]Remove the website build framework from the docs directory
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

1. Remove the website build framework from the docs directory
The current build framework is in the doris-website repo
2. Remove the build pipeline docs build

### What You Expected?

no

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by hf200012**
# Proposed changes

Issue Number: close #11080

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .github/workflows/build-extension.yml [hunk 1]
```diff
@@ -54,8 +54,3 @@ jobs:
     - name: Build broker
       run: |
         cd fs_brokers/apache_hdfs_broker/ && /bin/bash build.sh
-
-    - name: Build docs
-      run: |
-        cd docs &&  /bin/bash build_help_zip.sh && export NODE_OPTIONS="--max-old-space-size=8192" && npm install && npm run build
```
Comment (morningman) [inline .github/workflows/build-extension.yml:60]: we still need `build_help_zip.sh`

File: docs/.vuepress/sidebar/en/docs.js [hunk 2]
```diff
@@ -1,1014 +0,0 @@
-/*
```
Comment (morningman) [inline docs/.vuepress/sidebar/en/docs.js:1]: I think we still need this sidebar?

Comment (hf200012) [reply docs/.vuepress/sidebar/en/docs.js:1]: solved

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**